cmd/security-agent/app/config.go

```diff
@@ -0,0 +1,80 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build kubeapiserver
+
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	SecurityAgentCmd.AddCommand(configCommand)
+}
+
+var configCommand = &cobra.Command{
+	Use:   "config",
+	Short: "Print the runtime configuration of a running security agent",
+	Long:  ``,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		if flagNoColor {
+			color.NoColor = true
+		}
+
+		// we'll search for a config file named `datadog.yaml`
+		config.Datadog.SetConfigName("datadog")
```

Comment:
how does that work for the cluster agent?

Reply:
Cluster agent has its own config command 😄

Reply:
I forgot about it 😅